Ly/sync triggers (#926)
GSA-TTS/FAC: docs/branching.md

```

### Steps:
1. Start a branch from `main` for new work commit changes. When your work is ready, rebase to `main` and clean your commit history. When acceptance criteria is met and tests pass, create a pull request that targets the `main` branch. The terraform plan for staging will be created as part of the PR. Tag one or more relevant people for review.
2. Branch is ready for review. The reviewer will test locally for acceptance criteria, readability, security considerations, and good testing practices. Don't review your own PR. Once the PR is merged into the `main` branch, the code will deploy to the dev environment after tests pass. Any changes to the management or dev spaces via terraform will be applied.
3. When its time to create a release for review, we will add those changes to `prod` via PR. Pushing to the `prod` branch will deploy the code to staging after tests pass.

> 4. The release is reviewed by stakeholders. When it is ready for production, an OROS approver will add a version tag starting with "v". Any changes that were directly on the `prod` branch need to be merged back into the `main` branch with a PR. The tag will trigger an automated deploy to production. There is a GitHub rule to enforce that only "maintainer" role are allowed to add tags starting with "v."

Reviewer: ```suggestion
4. The release is reviewed by stakeholders. When it is ready for production, an OROS approver will add a version tag starting with "v". The tag will trigger an automated deploy to production. There is a GitHub rule to enforce that only "maintainer" role are allowed to add tags starting with "v." 
5. Any changes that are made directly on the `prod` branch will need to be merged back into the `main` branch with a PR. 
```

### Mechanics

> Our tests and deploys are kicked off from trigger files in [.github/workflows](https://github.com/GSA-TTS/FAC/blob/main/.github/workflows) that then call out the approiate scripts.

Reviewer: ```suggestion
Our tests and deploys are kicked off from trigger files in [.github/workflows](https://github.com/GSA-TTS/FAC/blob/main/.github/workflows) that then call out the appropriate scripts.
```

### Tips

> - Add initials to your branch name so you can tell where a branch started.

Reviewer: ```suggestion
- To help other know who to talk to about a branch, add initials to your branch name.
```

> - Add the ticket number to the branch name.

Reviewer: ```suggestion
- To help other understand the context for a branch, include the ticket number in the branch name.
```

> - DO NOT squash merge PRs to the `prod` branch or PRs from the `prod` branch back into `main`.

Reviewer: ```suggestion
- To minimize conflicts, DO NOT squash merge PRs to the `prod` branch or PRs from the `prod` branch back into `main`.
```
